### PR TITLE
grammar fixes

### PR DESCRIPTION
**Pull request by cjyetman**

### CODE REVIEW

**Review by jacobvjk — APPROVED** [review]

**Review by jdhoffa — APPROVED** [review]

**Comment by jdhoffa** [comment]
@cjyetman NB: maybe don't merge this yet? and instead merge all of these into a single "cookbook grammar review" PR

Otherwise we may end up with a git log of 100 "grammar fix" PRs 😊

File: vignettes/cookbook_preparatory_steps.Rmd [hunk 1]
```diff
@@ -210,7 +210,7 @@ cat(utils::packageDescription("pacta.multi.loanbook")[["Suggests"]])
...
+The most common ways to install R packages are via CRAN or GitHub. Public institutions often have restrictions on the installation of packages from GitHub, so you may need to install the package from CRAN. In some cases, your institution may mirror CRAN in their internal application registry, so you may need to install the package from there. Should you have any issues with the installation from the internal application registry, it is best to reach out to your IT department. If you cannot obtain the package through/in/with any of these ways, please reach out to the package maintainers directly for exploring other options.
```
Comment (cjyetman) [inline vignettes/cookbook_preparatory_steps.Rmd]: we should choose one of these, or suggest something better

**Comment by cjyetman** [comment]
all grammar fixes for preparatory steps are now combined here

**Comment by cjyetman** [comment]
FYI `pak` is not installing correctly suddenly in the Windows 4.1 CI  for some reason

File: vignettes/cookbook_preparatory_steps.Rmd (hunk 1 above)
Comment (jdhoffa) [reply vignettes/cookbook_preparatory_steps.Rmd]: in

Comment (jdhoffa) [reply vignettes/cookbook_preparatory_steps.Rmd]: After this change, LGTM

Comment (cjyetman) [reply vignettes/cookbook_preparatory_steps.Rmd]:
```suggestion
The most common ways to install R packages are via CRAN or GitHub. Public institutions often have restrictions on the installation of packages from GitHub, so you may need to install the package from CRAN. In some cases, your institution may mirror CRAN in their internal application registry, so you may need to install the package from there. Should you have any issues with the installation from the internal application registry, it is best to reach out to your IT department. If you cannot obtain the package in any of these ways, please reach out to the package maintainers directly for exploring other options.
```

**Review by jdhoffa — APPROVED** [review]

**Review by jacobvjk — APPROVED** [review]

**Pull request merged**